exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -364,6 +371,45 @@ def _extract_tags_from_dict(self, tags_dict):
             tags[attribute_key] = value
         return tags
 
+    def _extract_tag_value_string_from_sequence(self, sequence):
+        if not sequence:
+            return None
+
+        if self.max_tag_value_length == 1:
+            return None
+
+        tag_value_elements = []
+        running_string_length = (
+            2  # accounts for array brackets in output string
+        )
+        defined_max_tag_value_length = self.max_tag_value_length > 0
+
+        if isinstance(sequence, (list, tuple, range)):
```

Comment:
Looks like this can also be handled in the if on L375. I think it is safe to assume this private method will be called with the correct type.

Also, why not support all types of sequences, like is checked on L358 (e.g. some user defined sequence)?

Reply:
agree - reworked to allow all Sequence types and simplified type checking.